src/card.tsx

```diff
@@ -67,7 +67,7 @@ const CardContent: React.FC<CardContentProps> = ({
                                         </Text1>
                                     </Box>
                                 )}
-                                <Text4 as="h1" light>
```

Comment:
you must re-generate affected screenshot tests

from the repo root:
    yarn test-acceptance -u

then commit and push the result